src/java_bytecode/java_bytecode_language.h

```diff
@@ -10,13 +10,18 @@ Author: Daniel Kroening, kroening@kroening.com
 #define CPROVER_JAVA_BYTECODE_JAVA_BYTECODE_LANGUAGE_H
 
 #include <util/language.h>
+#include <util/cmdline.h>
 
 #include "java_class_loader.h"
 
+#define MAX_NONDET_ARRAY_LENGTH_DEFAULT 5
```

Comment:
Comment for future improvement: This does not seem the ideal place for a magic number.